src/main/java/io/swagger/model/Client.java

```diff
@@ -39,25 +38,13 @@
    * Gets or Sets race
    */
   public enum RaceEnum {
-    WHITE("White"),
-    
-    BLACK("Black"),
-    
-    HISPANIC("Hispanic"),
-    
-    OTHER("Other"),
-    
+    AMERICAN_INDIAN_OR_ALASKA_NATIVE("American Indian or Alaska Native"),
     ASIAN("Asian"),
-    
-    AMERICAN_INDIAN("American Indian"),
-    
-    BIRACIAL("Biracial"),
-    
-    WHITE_HISPANIC_OR_LATINO_("White [Hispanic or Latino]"),
-    
-    WHITE_BLACK_HISPANIC_OR_LATINO_("White/Black [Hispanic or Latino]"),
-    
-    UNKNOWN("Unknown");
+    BLACK_OR_AFRICAN_AMERICAN("Black or African American"),
+    HISPANIC_OR_LATINO("Hispanic or Latino"),
+    NATIVE_HAWAIIN_OR_OTHER_PACIFIC_ISLANDER("Native Hawaiian or Other Pacific Islander"),
```

Comment:
Hawaiian is spelled wrong.

HAWAIIN -> HAWAIIAN

Reply:
Good catch, this has been fixed.